matdeeplearn/trainers/base_trainer.py

```diff
@@ -123,17 +129,22 @@ def from_config(cls, config):
             test_loader=test_loader,
             loss=loss,
             max_epochs=max_epochs,
+            max_checkpoint_epochs=max_checkpoint_epochs,
             identifier=identifier,
             verbosity=verbosity,
+            save_dir=save_dir,
+            checkpoint_dir=checkpoint_dir,
         )
 
     @staticmethod
     def _load_dataset(dataset_config):
         """Loads the dataset if from a config file."""
         dataset_path = dataset_config["pt_path"]
-        target_index = dataset_config.get("target_index", 0)
 
-        dataset = get_dataset(dataset_path, target_index)
+        dataset = get_dataset(
+            dataset_path,
+            transform_list=dataset_config.get("transforms", []),
```

Comment:
If you decide that `GetY` is needed in order to avoid errors, then I think this should instead be
```suggestion
            transform_list=dataset_config.get("transforms", ["GetY"]),
```